README.md

```diff
@@ -339,31 +343,6 @@ ews.run(ewsFunction, ewsArgs, ewsSoapHeader)
 
 ```
 
-#### Use Encrypted Credentials for NTLM:
-This allows you to persist their password as separate hashes instead of as plain text.
-This utilizes the [options](https://github.com/SamDecrock/node-http-ntlm#options) available to the underlying NTLM lib.
-[Here](https://github.com/SamDecrock/node-http-ntlm#pre-encrypt-the-password) is an example from its README.
```

Comment:
You should replace this old documentation with a reference to the docs for the new library you're using, so that users understand what options to pass in (e.g. they wouldn't know about an option like `workstation` unless it was documented here).